model/src/main/scala/za/co/absa/atum/model/types/Measurement.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2024 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.model.types
+
+import za.co.absa.atum.model.ResultValueType
+import za.co.absa.atum.model.dto.MeasurementDTO
+
+trait Measurement {
```

Comment:
Same question as above - should this be in _Model_ or only in _Reader_?

Reply:
probably same answer - if this is only gonna be used in Reader, then it should be in Reader. If it's common enough to be also used in Agent, then I'd say let's keep it in model/.